applications/StructuralMechanicsApplication/tests/test_rve_analytic.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import print_function, absolute_import, division
+import KratosMultiphysics
+import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.kratos_utilities as kratos_utilities
+import RVEAnalysis
+
+
+class TestRVESimplestTest(KratosUnittest.TestCase):
+
+    def test_rve_computation_block_version(self):
+        with open("rve_test/smallest_rve_test_parameters.json", 'r') as parameter_file:
+            parameters = KratosMultiphysics.Parameters(parameter_file.read())
+
+        parameters["solver_settings"]["block_builder"].SetBool(True)
+        parameters["solver_settings"]["multi_point_constraints_used"].SetBool(True)
+
+        self._aux_rve_computation(parameters)
+
+    #@KratosUnittest.skip("Reactions and displacements not 100% accurate")
+    def test_rve_computation_elimination_version(self):
+        with open("rve_test/smallest_rve_test_parameters.json", 'r') as parameter_file:
+            parameters = KratosMultiphysics.Parameters(parameter_file.read())
+
+        parameters["solver_settings"]["block_builder"].SetBool(False)
+        parameters["solver_settings"]["multi_point_constraints_used"].SetBool(True)
+
+        self._aux_rve_computation(parameters)
+
+    def _aux_rve_computation(self, parameters):
+
+        model = KratosMultiphysics.Model()
+        simulation = RVEAnalysis.RVEAnalysis(model, parameters)
+        simulation.Run()
+
+        model_part = model["Structure.computing_domain"]
+
+        # Compare C
+        Cestimated = model_part.GetValue(StructuralMechanicsApplication.ELASTICITY_TENSOR)
+
+        Canalytic = KratosMultiphysics.Matrix(6, 6)
+        Canalytic.fill(0.0)
+        E = 1e6
+        nu = 0.3
+        l = E*nu/((1+nu)*(1-2*nu))
+        G = E/(2.0*(1.0+nu))
+        Canalytic[0, 0] = l+2*G
+        Canalytic[0, 1] = l
+        Canalytic[0, 2] = l
+
+        Canalytic[1, 0] = l
+        Canalytic[1, 1] = l+2*G
+        Canalytic[1, 2] = l
+
+        Canalytic[2, 0] = l
+        Canalytic[2, 1] = l
+        Canalytic[2, 2] = l+2*G
+
+        Canalytic[3, 3] = G
+        Canalytic[4, 4] = G
+        Canalytic[5, 5] = G
+
+        for i in range(0, Cestimated.Size1()):
+            for j in range(0, Cestimated.Size2()):
+                self.assertAlmostEqual(
+                    abs(Cestimated[i, j] - Canalytic[i, j])/(l+2*G), 0.0, 5)
+
+        if not parameters["rve_settings"]["print_rve_post"].GetBool():
```

Comment:
this check is not strictly needed, would also work without

Reply:
The point is that if i want the output i need the test not to delete the outputs...

Reply:
i leave the check since i wish to be able to actually see the output if i set the print_rve_post to true

Reply:
ok I see

I still have a suggestion for you: From what I understand you always print the output, even if you don't want to see it (in 99% of the cases this test is run)
=> this is time consuming
=> I recommend to do it the other way round, explicitly only adding the printing of output if you want it, take a look what I am doing [here](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/MeshMovingApplication/tests/test_ale_fluid_solver.py#L48-L101). This way the test won't get slowed down for nothing

Reply:
It is only been printed if the parameters["rve_settings"]["print_rve_post"] is True

Reply:
but why would you then need to delete the files at all? If you want to print them you would want to look at them right?

Reply:
ok I got it now, fixing it